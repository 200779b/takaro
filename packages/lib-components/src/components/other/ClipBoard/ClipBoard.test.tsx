import { ClipBoard } from '.';
import { render } from 'test-utils';

it('Should render <ClipBoard />', () => {
  const { container } = render(<ClipBoard text="ClipBoard text" />);
  expect(container).toMatchSnapshot();
});
