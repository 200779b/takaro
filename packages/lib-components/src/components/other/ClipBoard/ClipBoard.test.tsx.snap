// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render <ClipBoard /> 1`] = `
.c0 {
  position: relative;
  padding: 0;
  outline: 0;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
}

.c0 input {
  width: 200px;
  padding: 8px 30px 8px 8px;
  border: 1px solid #d3d3d3;
  color: #d3d3d3;
  cursor: default;
}

.c0 input:hover {
  cursor: default;
}

.c1 {
  position: absolute;
  background-color: white;
  top: 7px;
  right: 7px;
}

.c1 svg {
  fill: #d3d3d3;
  stroke: #d3d3d3;
  cursor: pointer;
}

<div>
  <div
    class="c0"
    role="button"
    tabindex="-1"
  >
    <input
      aria-describedby="clipboard-input-error"
      aria-invalid="false"
      readonly=""
      type="text"
      value="ClipBoard text"
    />
    <div
      class="c1"
    >
      <svg
        fill="currentColor"
        height="20"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 1024 1024"
        width="20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M832 64H296c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h496v688c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V96c0-17.7-14.3-32-32-32zM704 192H192c-17.7 0-32 14.3-32 32v530.7c0 8.5 3.4 16.6 9.4 22.6l173.3 173.3c2.2 2.2 4.7 4 7.4 5.5v1.9h4.2c3.5 1.3 7.2 2 11 2H704c17.7 0 32-14.3 32-32V224c0-17.7-14.3-32-32-32zM350 856.2L263.9 770H350v86.2zM664 888H414V746c0-22.1-17.9-40-40-40H232V264h432v624z"
        />
      </svg>
    </div>
  </div>
</div>
`;
